### PR TITLE
E2E tests: Fix Slack notification for scheduled jobs

### PR DESCRIPTION
**Pull request by adimoldovan**
#### Changes proposed in this Pull Request:

The `github.event` object used to get context information seems to be (almost) empty for scheduled jobs. This PR changes the `github.event` with the top level `github` context object that provides all the required info for all types of events that trigger the tests.

#### Jetpack product discussion
1200222342137241-as-1200352706466069

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* CI green
* Slack notification is sent

### CODE REVIEW

File: .github/workflows/e2e-tests.yml
Comment (brbrr) [inline .github/workflows/e2e-tests.yml:147]: I'm curious to know why you moved the step here.

**Review by brbrr — APPROVED** [review]
The code looks good! Let's wait for CI to finish before merging 👍

File: .github/workflows/e2e-tests.yml
Comment (adimoldovan) [reply .github/workflows/e2e-tests.yml:147]: I didn't want a failure in the `testspace` step to trigger a test failure notification.

**Pull request merged**
